pkg/version/version_go1_12.go

```diff
@@ -1,3 +1,4 @@
+//go:build go1.12
 // +build go1.12
```

Comment:
Does this obviate line 2, or is it additional?

Reply:
Ah, GitHub Copilot to the rescue.

<img width="478" alt="image" src="https://github.com/segmentio/ctlstore/assets/1111441/7b9e333f-4ec0-47a8-be22-5c63f4ffbdac">


I wonder if we can get rid of the older-style expression.  I guess it might break if one of our client applications is still building with Go version less than 1.17.

Reply:
Precisely

Reply:
I'm not sure I've seen anything as old as 1.17, but probably better safe than sorry